from libs.exception.bot_exception import BotException


class StorageException(BotException):
    pass
